docs/roman/ramp_fitting/description.rst

```diff
@@ -193,6 +193,114 @@ Upon successful completion of this step, the status attribute ramp_fit will be s
 to "COMPLETE".
 
 
+Jump Detection
+==============
+
+If the uneven-ramp jump detection algorithm is turned on (the default), the ramp fitting
+algorithm is then run iteratively on a "queue" (list) of ramps. The queue is initialized
+with the ramp(s) (multiple ramps in the case of a dq flag occuring in the middle of a ramp).
+Then the jump detection algorithm picks a ramp, say :math:`[R_0, \dots, R_M]`, out of the
+queue and runs the ramp fitting algorithm on it. It then checks the resulting ramp for jumps.
+If a jump is detected, then two sub-ramps are created from the passed in ramp which exclude
+the resultants predicted to be affected by the jump. These sub-ramps are then added to the
+queue. This process continues until the queue is empty.
+
+.. note::
+
+   It may not always be possible to create two sub-ramps around the resultants predicted to
+   be part of a jump. For example if these jump resultants include the first, second, second-to-last,
+   or last resultant of the ramp then it is not possible to create two meaningful sub-ramps, as one
+   cannot run the ramp fitting algorithm on a ramp with zero or only one resultant. Therefore, in
+   these cases, only one sub-ramp is created and added to the queue.
+
+The method use for determining if and where a jump occurs is divided into two parts. First,
+a *statistic*, :math:`S` and possible jump resultants are determined for the fitted ramp.
+Then the statistic is compared against a threshold function, :math:`S_{\text{threshold}}` to determine
+if a jump has occurred.
+
+
+Statistic and Possible Jump Resultants
+++++++++++++++++++++++++++++++++++++++
+
+The statistic used to determine if a jump has occurred in the ramp, :math:`[R_0, \dots, R_M]`,
+is computed from a list of statistics computed for each *single* and *double-difference* of
+the resultants in the ramp.  By single-difference we mean the difference between two adjacent
+resultants in the ramp, while double-difference refers to the difference between a resultant
+and a resultant two steps away (the resultant adjacent to a resultant adjacent to the resultant
+in question).
```

Comment:
Let's add some comment saying: this follows Sanjib et al. (2023) (in preparation).